binder/jupyter_config.py

```diff
@@ -43,3 +43,11 @@
 }
 
 c.ServerApp.log_level = logging.DEBUG
+
+c.LabApp.dev_mode = True
+c.LabApp.collaborative = True
```

Comment:
The flag is not needed as there is a need for explicitly installing an additional package.

```suggestion
```

> And for now it will fail as while setting `collaborative`, the required test is failing: fix is there - https://github.com/jupyterlab/jupyterlab/pull/14047

Reply:
when it's released, we can fix it

Reply:
Sorry I wasn't clear - that flag can already be removed as the collaboration code has been removed from core. The flag only exists to inform user (familiar with that flag in 3.x) that it should be done by installing another extension now.

Reply:
so what's the right play: take out _all_ collaboration-related stuff now, and do all of it on a follow-on PR?

Reply:
removed